extensions/big-data-cluster/package.json

```diff
@@ -1,94 +1,93 @@
-                "yaml"
-              ],
-              "type": "string",
-              "description": "Output format for Kubernetes specs. One of 'json' or 'yaml' (default)."
-            }
-          },
-          "default": {
-            "mssql-bdc.namespace": "",
-            "mssql-bdc.kubectl-path": "",
-            "mssql-bdc.kubeconfig": "",
-            "mssql-bdc.knownKubeconfigs": []
-          }
-        }
-      }
-    },
-    "commands": [
-      {
-        "command": "mssql.cluster.create",
-        "title": "Create SQL Server big data cluster",
-        "category": "SQL Server"
-      }
-    ]
-  },
-  "dependencies": {
-    "vscode-nls": "^3.2.1",
-    "download": "^6.2.5",
-    "shelljs": "^0.8.3"
-  },
-  "devDependencies": {
-    "mocha-junit-reporter": "^1.17.0",
-    "mocha-multi-reporters": "^1.1.7"
-  }
+	"name": "big-data-cluster",
+	"displayName": "%displayName%",
+	"description": "%description%",
+	"version": "0.0.1",
+	"publisher": "Microsoft",
+	"preview": true,
+	"license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt",
+	"icon": "images/sqlserver.png",
+	"engines": {
+		"vscode": "*",
+		"azdata": "*"
+	},
+	"activationEvents": [
+		"*"
+	],
+	"forceReload": true,
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Microsoft/azuredatastudio.git"
+	},
+	"main": "./out/extension",
+	"contributes": {
+		"dataExplorer": {
+			"sqlBigDataCluster": [
+				{
+					"id": "sqlBigDataCluster",
+					"name": "%explorer.title%"
+				}
+			]
+		},
+		"menus": {
+			"view/title": [
+				{
+					"command": "bigDataClusters.command.addController",
+					"when": "view == sqlBigDataCluster",
+					"group": "navigation"
+				}
+			],
+			"view/item/context": [
+				{
+					"command": "bigDataClusters.command.deleteController",
+					"when": "viewItem == bigDataClusters.itemType.ControllerNode",
+					"group": "navigation@1"
+				}
+			]
+		},
+		"configuration": {
+			"type": "object",
+			"title": "%config.title%",
+			"properties": {
+				"bigDataClusterControllers.controllers": {
+					"type": "array"
+				}
+			}
+		},
+		"commands": [
+			{
+				"command": "bigDataClusters.command.addController",
+				"title": "%command.addController.title%",
+				"icon": {
+					"light": "resources/light/add.svg",
+					"dark": "resources/dark/add_inverse.svg"
+				}
+			},
+			{
+				"command": "bigDataClusters.command.deleteController",
+				"title": "%command.deleteController.title%",
+				"when": "viewItem == bigDataClusters.itemType.ControllerNode"
+			},
+			{
+				"command": "bigDataClusters.command.refreshController",
+				"title": "%command.refreshController.title%",
+				"icon": {
+					"light": "resources/light/refresh.svg",
+					"dark": "resources/dark/refresh_inverse.svg"
+				}
+			}
+		]
+	},
+	"dependencies": {
+		"request": "^2.88.0",
+		"vscode-nls": "^4.0.0"
+	},
+	"devDependencies": {
+		"@types/mocha": "^5.2.5",
+		"@types/node": "^8.0.24",
+		"mocha": "^5.2.0",
```

Comment:
>"mocha": "^5.2.0", [](start = 2, length = 18)

I don't see any tests so you shouldn't need any of the mocha-related stuff. That being said seeing some tests would be an excellent idea

Reply:
👍 on tests

Reply:
test will be added.